package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ion-study-template",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w",
+    "lint": "eslint . && web-ext lint",
+    "test": "npm run test-addon"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^14.0.0",
+    "@rollup/plugin-node-resolve": "^10.0.0",
```

Comment:
Do we need rollup from the beginning?

Reply:
Yes, as we discussed in our 1:1 (after this comment was made), yes, we'll need that since the "ion.js" library will be delivered using NPM and bundled.